common-practices-tools/security/yubikey.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Using your YubiKey
+---
+
+Notes on installing and setting up your _YubiKey_ for various platforms and applications.
+
+## Introduction
+
+The YubiKey is a hardware device manufactured by Yubico that provides a hardware "second factor" enabling true multi-factor authentication: something you know (your password) and something you have (your YubiKey). It enables you to easily and securely log in to accounts by emitting one-time passwords or using a FIDO-based public/private key pair generated by the device.
+
+### Operation
+
+-   Simply plug it into an unused USB port.
+-   During certain types of authentication you will be prompted on screen to press the inset copper button marked with and (often lit) "Y".
+
+### Security Hints
+
+-   If you trust your environment (like at home) you can keep the YubiKey near or even plugged into your computer.
+-   In low trust environments (coffee shops, hotel rooms, etc.) keep your YubiKey with you at all times (in a pocket or purse). If your computer is compromised, it won't be accessible without the YubiKey that you have on you.
+-   Do not use SMS text messages for two-factor authentication. Your TFA is only as strong as the weakest method you have configured.
+
+## Enable YubiKey TFA for applications
+
+See "[Let's get your YubiKey to work](https://yubico.com/start)" (from Yubico) on how to use MFA with: [Gmail](https://www.yubico.com/why-yubico/for-individuals/gmail-for-individuals), [LastPass](https://www.yubico.com/works-with-yubikey/catalog/lastpass-premium-and-families/), [GitHub](https://www.yubico.com/works-with-yubikey/catalog/github/) and many other services.
+
+_See also: [How to Secure Your Google, Dropbox, and GitHub Accounts With a U2F Key](http://www.howtogeek.com/232360/how-to-secure-your-google-dropbox-and-github-accounts-with-a-u2f-key/)_
```

Comment:
First time using this acronym ("U2F") -- maybe should call it "a hardware key" or "your Yubikey"?

Reply:
Updated to just reference Yubikey.